### DISCUSSION TITLE
A problem with RAM availability for indexing

### DISCUSSION THREAD

**Discussion by talcohenAG** [alexdobin/STAR#1490] [Q&A]
Hi,

I wish to create a human genome index (genomeGenerate) with the newest STAR version (2.7.10a), but it seems I don't have enough RAM available.
I tried reducing number of threads, as well as one of the solutions suggested here (--genomeSAsparseD 3 --genomeSAindexNbases 12 --limitGenomeGenerateRAM 1500000000), but if I use the --limitGenomeGenerateRAM parameter, I get this error:

EXITING because of FATAL PARAMETER ERROR: limitGenomeGenerateRAM=1500000000is too small for your genome
SOLUTION: please specify --limitGenomeGenerateRAM not less than 8369034848 and make that much RAM available

Is there anything else I could try?
Alternatively, is there anywhere I could download the genome index (for ENSEMBL GRCh38) for this current version?

Thanks,
Tal

**Comment by alexdobin** [comment]
Hi Tal,

I think you are missing one 0 in ```--limitGenomeGenerateRAM 15000000000``` i.e. 15GB=15,000,000,000

Cheers
Alex

**Reply by talcohenAG** [reply]
Thanks Alex,
I had a problem with free space and free memory, so I ended up downloading an indexed genome.
